src/oss-reproducible/Strategies/PackageIgnoreList.txt

```diff
@@ -0,0 +1,25 @@
+﻿# All files
```

Comment:
Recommend adding a comment to the top here to explain the format for future users.